paper/comoving-followup.tex

```diff
@@ -323,18 +324,19 @@ \subsection{Observations and data reduction}\label{sec:reduction}
 solution on a per-object basis.
 
 We observed a total of 765 unique sources over 5 nights from (UT) 2017-03-11 to
-2017-03-15, corresponding to 323 unique star pairs and $\approx 120$
```

Comment:
I think it's well-understood that the pairs we discuss through out are pairs of stars and nothing else, so I find 'star pair' a little verbose and awkward especially since it's not consistent either. I didn't go so far as to correct all of them. Something to consider.